source/docs/appendices/migration-guides.md

```diff
@@ -32,6 +37,35 @@ The environment variable `FRONT_COMMERCE_PRISMIC_URL` has been removed. Please u
 + FRONT_COMMERCE_PRISMIC_REPOSITORY_NAME=my-repo
 ```
 
+#### Improved the `registerRoutableType` method
+- A new required property has been added `path` for dynamic routes. <br /> Examples: `/:uid`, `/:lang/:uid`, `/:category*/:uid`, `/:section/:category?/:uid`.
+<blockquote class="info">
+**ProTip :** You can use the online [express-route-tester@2.0.0](http://forbeslindesay.github.io/express-route-tester) to test your paths.
+</blockquote>
+- The method now allows you to register a new routes with the prismic client using the `withPrismicRoutes` property. This will allow prismic to redirect previews for document types to the correct path, and it will also allow the url property to be resolved in the documents.
```

Comment:
```suggestion
- The method now allows you to register a new route with the prismic client using the `withPrismicRoutes` property. This will allow prismic to redirect previews for document types to the correct path, and it will also allow the url property to be resolved in the documents.
```